docs/app-dev-context/connecting-temporal-client.md

```diff
@@ -0,0 +1,13 @@
+When connecting to a Temporal Client, you must provide the address and port number, if the SDK doesn't default to an address you want to connect to.
```

Comment:
The connection is from the Client to the Cluster.

Might want to change this section to "connect-to-a-cluster"

Or be be verbose - "connect a temporal client to a temporal cluster"